modules/group/variables.tf

```diff
@@ -22,6 +22,11 @@ variable "inline_policies" {
     policy = string
   }))
   default = []
+
+  validation {
+    condition     = length(var.inline_policies) > 0 ? sum([for item in var.inline_policies : length(item.policy)]) <= 5120 : true
```

Comment:
Unfortunately, need to check the list length because `sum()` will fail if it gets an empty list. 🙄

Reply:
Also, need to use the ternary operator in order to short-circuit the `sum()` expression. If we used `||` then _both_ side would evaluate (and `sum()` would fail)... i.e. this doesn't work either:

```
length(var.inline_policies) == 0 || sum([for item in var.inline_policies : length(item.policy)]) <= 5120
```